test/run_test.py

```diff
@@ -85,7 +85,7 @@ def skip_test_p(name: str) -> bool:
         'bottleneck_test',
         'custom_backend',
         'custom_operator',
-        'fx',        # executed by test_fx.py
+        'fx/',        # executed by test_fx.py
```

Comment:
Can you please explain why those changes are needed?

Reply:
> 

in dir caffe2/test, there are dir with name "fx", "fx_acc", "fx2trt", this change is to only block test under dir caffe2/test/fx/...